lib/src/cli.rs

```diff
@@ -576,15 +578,46 @@ pub(crate) async fn get_storage() -> Result<crate::store::Storage> {
 }
 
 #[context("Querying root privilege")]
-pub(crate) fn require_root() -> Result<()> {
-    let uid = rustix::process::getuid();
-    if !uid.is_root() {
-        anyhow::bail!("This command requires root privileges");
-    }
-    if !rustix::thread::capability_is_in_bounding_set(rustix::thread::Capability::SystemAdmin)? {
-        anyhow::bail!("This command requires full root privileges (CAP_SYS_ADMIN)");
-    }
+pub(crate) fn require_root(is_container: bool) -> Result<()> {
+    ensure!(
+        rustix::process::getuid().is_root(),
+        match is_container {
+            true =>
+                "The user inside the container from which you are running this command must be root",
+            false => "This command must be executed as the root user",
+        }
```

Comment:
I'd write this 
`if is_container { "The user inside the container from which you are running this command must be root" } else {  "This command must be executed as the root user" }`